pages/index.js

```diff
@@ -14,29 +14,30 @@ import HelpingArea from "../src/components/HelpingArea";
 import TeamArea from "../src/components/TeamArea";
 import CallToAction from "../src/components/CallToAction";
 import CallToActionTwo from "../src/components/CallToActionTwo";
-
+import Error from 'next/error'
 import GET_STUDENTS_COUNT from '../src/graphql/query/students';
 
 const HomePage = ({ props }) => {
-  return (
-    <Layout pageTitle="ROSHANI - Light towards life">
-      <NavOne />
-      <SliderOne />
-      <HiwArea />
-      <MixerArea studentsCount = { props.studentsCount } />
-      <CausesArea />
-      <HelpingArea />
-      <TeamArea />
-      <CallToAction />
-      <CallToActionTwo />
-      <Footer />
-    </Layout>
-  );
+  if (props && props.studentsCount) {
```

Comment:
@AAMENN This logic doesn't seem to be correct. We don't want entire home page to fail in loading, because of students count. Also we dont want to render any error message is props & props.studentsCount doesn't exist. These kind of errors should fail silently, without throwing any UI errors.